backupzip/backupzip.go

```diff
@@ -0,0 +1,61 @@
+package backupzip
+
+import (
+	"archive/zip"
+	"io"
+	"os"
+	"path/filepath"
+)
+
+func OutputZipBackupFile(fluidkeysDir, armoredPublicKey string, armoredPrivateKey string) (filename string, err error) {
+	filename = getZipFilename(fluidkeysDir)
+
+	backupZipFile, err := os.Create(filename)
+	if err != nil {
+		return "", err
```

Comment:
Can we prefix the error with something that offers a little more context in case of failing?